src/utils/forRender.js

```diff
@@ -0,0 +1,26 @@
+import Iso from 'iso'
+import * as Render from './Render'
+
+// XXX include the promise polyfill?
+
+export const withData = Render.withData
+
+export default function forRender(alt, Component, props) {
+  // recycle state
+  alt.recycle()
+
+  /*eslint-disable */
+  if (typeof window === 'undefined') {
+    /*eslint-enable */
```

Comment:
Since you added `browser: true` in eslint configuration shouldn't you remove the enable/disable comments?